app/models/finance/billing_strategy.rb

```diff
@@ -374,10 +364,6 @@ def bill_and_charge_each(options = {})
 
         msg = "Failed to bill or charge #{name}(#{buyer_id}) of provider(#{provider_id}): #{exception.message}\n"
         error(msg, buyer)
-        airbrake(:error_message => msg,
```

Comment:
Although the method was called `airbrake`, it's been actually reporting to Bugsnag:

https://github.com/3scale/porta/tree/master/app/models/finance/billing_strategy.rb#L416

https://github.com/3scale/porta/tree/master/app/lib/system/error_reporting.rb#L20

Removing this call will change that. Is this really what we want?

Reply:
By the way, according to the second condition [`Rails.env.preview?`](https://github.com/3scale/porta/blob/c1fbcb787ff4b4d8f257a06a7a22705d0ec08563/app/models/finance/billing_strategy.rb#L415), errors will be sent to Bugsnag when in preview, but that seems to contradict [`error_reporting_stages`](https://github.com/3scale/porta/blob/e2baf42638754496838ca5b579f62f72bd4bda24/config/initializers/bugsnag.rb#L5-L6).

Reply:
I agree. Let's `delegate :report_error, to: 'System::ErrorReporting'` and replace those `airbrake` calls to a simple `report_error`.